app/models/match.rb

```diff
@@ -40,7 +40,7 @@ def activate
   end
 
   def conclude
-    update(concluded_at: Time.zone.now)
+    update(concluded_at: Time.zone.now, send_conclusion_mail: false)
```

Comment:
Eh never mind, I forgot how things worked. But perhaps we should rename this method `abort` or something to better signal the intent?